<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_1p">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeMap">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayersSettings">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Players Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>520</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back to Main menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="map1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Map 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="map1picture">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/map1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="map2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Map 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Scin1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Player 1 Scin</string>
    </property>
   </widget>
   <widget class="QLabel" name="Scin1picture">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="map2picture">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/map1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Scin1_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Player 1 Scin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Scin2_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>160</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Player 2 Scin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Scin2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Player 2 Scin</string>
    </property>
   </widget>
   <widget class="QLabel" name="Scin1picture2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Scin2picture">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Scin2picture2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>pushButton_1p</zorder>
   <zorder>ChangeMap</zorder>
   <zorder>PlayersSettings</zorder>
   <zorder>Back</zorder>
   <zorder>map2</zorder>
   <zorder>Scin1</zorder>
   <zorder>map1</zorder>
   <zorder>map1picture</zorder>
   <zorder>Scin1picture</zorder>
   <zorder>map2picture</zorder>
   <zorder>Scin1_2</zorder>
   <zorder>Scin2_2</zorder>
   <zorder>Scin2</zorder>
   <zorder>Scin1picture2</zorder>
   <zorder>Scin2picture</zorder>
   <zorder>Scin2picture2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
